xmtp_mls/src/groups/mod.rs

```diff
@@ -441,6 +444,7 @@ where
                     .map(|m| m.from_ns),
             )
             .message_disappear_in_ns(opts.message_disappearing_settings.as_ref().map(|m| m.in_ns))
+            .should_publish_commit_log(should_publish_commit_log)
```

Comment:
@richardhuaaa updated to mark groups created from backup as `false` for `should_publish_commit_log`. If our client is in fact a super admin this should get updated in our db when we process a welcome for that group.